releasenotes/notes/marginal_counts_act_on_memory-0a9b58d0b95046dd.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Update the `marginal_counts` method to marginalize the memory field of the data.
```

Comment:
I think this could use a bit more detail explaining more about how this is new. A simple example demonstrating what this means to users would go a long way. You don't need a full working code example, but if you just say assuming the result had a single experiment and it contained memory of `[0x1, 0x2, 0x2, 0x3]` calling `marginal_counts()` with an example call and show how the memory list will be updated.